### DISCUSSION TITLE
What does the "Draw.io deployment setting actually do?

### DISCUSSION THREAD

**Discussion by zack-emmert** [outline/outline#7192] [Self Hosting] [answered]
I've been poking around the admin settings, and went to Integrations -> Self Hosted and found two URL settings I don't understand: One is for a self-hosted Draw.io instance, and the other is for a self hosted Grist instance.

 I don't know what changing either of these settings actually does. I've spun up a Draw.io container, and put the URL in the Outline settings, but nothing seems to have changed from an editor standpoint. I haven't messed with Grist, but I think I might if I can figure out what its corresponding Outline setting is for. I also can't find any notes about these settings in the Outline documentation.

**Comment by tommoor** [comment]
It doesn't do _a lot_, in the future neither of these will be needed – but right now it allows you to paste links from your self-hosted container and have them embed correctly as they would from the hosted service.

**Reply by keem14** [reply]
Hi @tommoor,
I have been very interested on self-hosting outline and embed some pages from a self-hosted grist into it.
I am currently using the latest outline version v1.0.1. 
Unfortunately I wasn't able to embed the self-hosted grist table using the /grist slash command.
But strangely I were able to embed grist the following ways:
- embed grist table from their "getgrist" website : https://docs.getgrist.com/ using the /grist slash command.
- embed grist table from a self-hosted grist using /embed slash command and tailing the URL with ?embed=true.

Am I messing something?

**Reply by keem14** [reply, marked as answer]
Never mind, I was going through the release changes and I have found the following change:
May 11 v0.84.0:
- "Self hosted" integrations settings were removed, you can embed any self-hosted apps through /embed command now.

So please allow me to use this opportunity to thank you and the other contributors for your efforts of making this wonderful project. : )
